db/migrate/20200923150158_create_measurement_types.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMeasurementTypes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :measurement_types do |t|
+      t.string :name # count || length
+      t.string :unit # integer || cm
+      t.references :organization
```

Comment:
should there be a unique index here `t.index [:name, :organization]`

Reply:
Possibly, I suppose it is possible to have a measurement performed in two units, so I guess some clarification around how it's used. If, say, "length" were always "cm" then yes, I think you could have one uniquely named measurement type per org. If there could be "length" - "cm", "length" - "mm" that would be my concern.

Reply:
That makes sense. Thanks for clarifying. Perhaps there should be a unique index for all three columns?

Reply:
Yeah, that would almost certainly be safe to assume in my opinion, and the db level constraint is always a nice extra safeguard.